workspaces/github-actions/app-config.yaml

```diff
@@ -9,6 +9,23 @@ app:
     - entity-card:catalog/links:
         config:
           filter: kind:component has:links
+    - entity-content:github-actions/entity
+    - entity-card:github-actions/workflow-runs:
+        config:
+          props:
+            variant: gridItem
```

Comment:
Do we need to expose this as configuration at all? thinking we should be able to set a sensible default here instead